ccdproc/tests/test_image_collection.py

```diff
@@ -642,16 +642,15 @@ def test_keyword_order_is_preserved(self, triage_setup):
         assert ic.keywords == ['file'] + keywords
 
     def test_sorting(self, triage_setup):
-        collection = image_collection.ImageFileCollection(location=triage_setup.test_dir,
-                                             keywords=['imagetyp',
-                                                       'filter',
-                                                       'object'])
+        collection = image_collection.ImageFileCollection(
+            location=triage_setup.test_dir,
+            keywords=['imagetyp', 'filter', 'object'])
 
         all_elements = []
         for hdu, fname in collection.hdus(return_fname=True):
             all_elements.append((str(hdu.header), fname))
         # Now sort
-        collection.sort(keys=['filter', 'object'])
+        collection.sort(keys=['imagetyp', 'object'])
```

Comment:
Recently `np.ma.argsort` was changed, it didn't compare masked entries before (which are `None` because no header contains a `filter`) but now they do. However `None`s can't be compared.